src/shared/dropdown-list/dropdown-list.tsx

```diff
@@ -307,6 +336,11 @@ export const DropdownList = <T, V>({
                                     hasExceededContainer(item)
                                         ? renderTruncatedText(item)
                                         : getOptionLabel(item)}
+                                    {secondaryLabelExtractor && (
+                                        <div className="secondary-label">
```

Comment:
Should use a span instead of div. And also to create a styled component instead of using `className`